packages/block-library/src/legacy-widget/index.php

```diff
@@ -5,6 +5,63 @@
  * @package WordPress
  */
 
+/**
+ * Returns the result of rendering a widget having its instance id.
+ *
+ * @param string $id Widget id.
+ *
+ * @return string Returns the rendered widget as a string.
+ */
+function render_widget_by_id( $id ) {
+	// Code extracted from src/wp-includes/widgets.php.
```

Comment:
Let's mention which method this code comes from (`dynamic_sidebar()`, yeah?).

When this code is merged into Core, I'd like to see this part of `dynamic_sidebar()` split out into its own method. We'll need to remember to do this somehow e.g. a Trac ticket or a `TODO` comment here.